wizeline/wize-q-front
# Fix class name and remove unused css

#### What does this PR do?
- Change the class name that was breaking the style.
- Remove some css classes that were not being used.
- Changes the ugly `X` with a fancy `Logout` text.

#### Where should the reviewer start?
`src/components/App.js`

#### How should this be manually tested?
1. Log in.
2. Try to post a question as you user, you'll see the style is not broken anymore.
3. Try to logout. Same thing.
4. Check out the cool new logout button. 😎 

#### What are the relevant tickets?
Closes [WIP-306](https://wizeline.atlassian.net/browse/WIP-306).

#### Screenshots :camera:
**Logout modal**
<img width="611" alt="screen shot 2017-07-05 at 9 26 44 am" src="https://user-images.githubusercontent.com/15201480/27869247-f1fd217c-6164-11e7-94d2-c70248f0a598.png">

**Main page (with new logout button style)**
<img width="1352" alt="screen shot 2017-07-05 at 9 26 57 am" src="https://user-images.githubusercontent.com/15201480/27869246-f1fbe1fe-6164-11e7-8bad-9dfcce791922.png">
